tests/certification/pubsub/azure/eventhubs/README.md

```diff
@@ -0,0 +1,82 @@
+# Azure Event Hubs Pubsub certification testing
+
+This project aims to test the Azure Event Hubs pubsub component under various conditions.
+
+## Test Plan
+
+### Pubsub Certification Tests
+  - Test partition key, in order processing single publisher/subscriber (done as part of conformance tests)
+  - Test single publisher / single subscriber
```

Comment:
This test is not created yet.

Reply:
This is being tested with along with entity management test.
- create topic-> publish messages-> subscribe messages-> check if the messages were received in order.

Reply:
Added description in form of comment.